graphdatascience/graph/graph_proc_runner.py

```diff
@@ -153,9 +153,12 @@ def list(self, G: Optional[Graph] = None) -> DataFrame:
         return self._query_runner.run_query(query, params)
 
     @client_only_endpoint("gds.graph")
-    def get(self, graph_name: str) -> Graph:
+    def get(self,
+            graph_name: str,
+            failIfMissing: bool = True,
```

Comment:
If this is `False` but there is no graph, we cannot return a Graph object. Instead, we should return `None`, I think.